client/state/utils/README.md

```diff
@@ -0,0 +1,18 @@
+Utils
+========
+
+A module for those little helpers that need to interact woth the store.
+
+# Actions
+Used in combination with the Redux store instance `dispatch` function, actions can be used in manipulating the current global state.
+
+## uniqueId( prefix )
+Meant as a replacement for _ and lodash "uniqueId" function.
+Returns unique numeric prefixed by prefix and increments internal state, so the next uniqueId will be different.
```

Comment:
I'm confused... the `uniqueId()` thunk returns a function and not an action object, and you are passing that function as an argument into the store instance's `dispatch()` method here which [expects an action object](http://rackt.org/redux/docs/api/Store.html#dispatch). So that doesn't appear valid to me.